packages/cli/src/services/CommandService.test.ts

```diff
@@ -84,6 +89,7 @@ describe('CommandService', () => {
         expect(loadedTree.length).toBe(4);
```

Comment:
Hi @abhipatel12 I just realized the test check is failing as I forgot to update in CommanService.test.ts a few expects from 4 to 5 since there are 5 commands now, can it still be adjusted before the merge or how do we deal with this?